### ISSUE TITLE
add a sprite editor

### ISSUE THREAD

**Issue by Labbo-Lab-AKA-Blueverse-Dog** [krzys-h/UndertaleModTool#354]

**Comment by BenjaminUrquhart** [comment]
I don't really think this is necessary. Just use mspaint or a pixel art editor such as piskel and the import graphics scripts.

**Comment by Labbo-Lab-AKA-Blueverse-Dog** [comment]
> I don't really think this is necessary. ### Just use mspaint or a pixel art editor such as piskel and the import graphics scripts.
thats the part that i wish i didn't need to do
but ig you're right
its a small price to pay for an amazing tool :D

**Comment by Grossley** [comment]
> please just do it all i want for christmas is a sprite editor built in so i don't have to download the sprite in undertalemodtool, open a new piskel tab, import the sprite, edit the sprite, go back to undertalemodtool, replace the sprite with the one i edited.

I agree with this issue, ideally a sprite editor should be implemented in the tool itself. However, it is low priority due to other options being available as said above.
